packages/hydrogen-edge-adapter/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@netlify/hydrogen-edge-adapter",
+  "version": "1.0.0",
+  "description": "Hydrogen/Remix Adapter for Netlify Edge Functions",
+  "types": "./build/src/index.d.ts",
+  "main": "./build/src/index.js",
+  "exports": {
+    ".": "./build/src/index.js",
+    "./config": "./build/src/defaultRemixConfig.js"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "build/src/**/*.d.ts.map",
```

Comment:
I know that this was probably copied over from the other package but I'd personally not publish source maps with our packages as they only add bloat without a benefit (people won't need them)

Reply:
Sounds fine to me!